TeamSnapSDK/SDK/CollectionJSON/TSDKCollectionQuery.h

```diff
@@ -18,5 +18,6 @@
 
 - (instancetype _Nullable)initWithJSONDict:(NSDictionary *_Nonnull)jsonDict;
 -(void)executeWithCompletion:(TSDKCompletionBlock _Nullable)completion;
+-(void)executeWithExtraParameters:(NSDictionary *_Nullable)extraParameters completion:(TSDKCompletionBlock _Nullable)completion;
```

Comment:
SUPER NIT DEF NOT BLOCKING: I forget off the top of my head but does this synthesize in swift to `execute(with extraParameters....` ? I would vote to keep the signature the same as the `executeWithCompletion` if possible, almost like it had a default param just for consistency

Reply:
This method is not being called in any swift file. But I believe it would be `execute(withExtraParameters ...)`